packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -26,7 +26,8 @@ import 'ticker_provider.dart';
 /// [GlowingOverscrollIndicator] generates [OverscrollIndicatorNotification]
 /// before showing an overscroll indication. To prevent the indicator from
 /// showing the indication, call [OverscrollIndicatorNotification.disallowGlow]
-/// on the notification.
+/// on the notification. In order to paint the glow at a different offset,
```

Comment:
Let's move this comment into a separate section that will discuss the options for manipulating the glow in custom scroll views. How about adding a `## Customizing Glow for Advanced Scroll Views` or something like that starting at line 37 below?

Reply:
Done.